### PR TITLE
Modify S2083(js): Fix code samples

### PR DESCRIPTION
**Pull request by loris-s-sonarsource**
## Review

A dedicated reviewer checked the rule description successfully for:

- [x] logical errors and incorrect information
- [x] information gaps and missing content
- [x] text style and tone
- [x] PR summary and labels follow [the guidelines](https://github.com/SonarSource/rspec/#to-modify-an-existing-rule)

### CODE REVIEW

**Review by gaetan-ferry-sonarsource — APPROVED** [review]
lgtm

File: rules/S2083/javascript/how-to-fix-it/node.adoc [hunk 1]
```diff
@@ -13,7 +13,7 @@ include::../../common/fix/code-rationale.adoc[]
 const path = require('path');
 
 function (req, res) {
-  const targetDirectory = "/data/app/resources/"
+  const targetDirectory = "/data/app/resources/";
   const userFilename = path.join(targetDirectory, req.query.filename);
 
   let data = fs.readFileSync(userFilename, { encoding: 'utf8', flag: 'r' }); // Noncompliant
```
Comment (egon-okerman-sonarsource) [inline rules/S2083/javascript/how-to-fix-it/node.adoc:19]: If we're fixing things here, maybe we can also import `fs`?

Comment (loris-s-sonarsource) [reply rules/S2083/javascript/how-to-fix-it/node.adoc:19]: done ! thanks

**Review by egon-okerman-sonarsource — APPROVED** [review]
LGTM 🎊

**Pull request merged**